main.py

```diff
@@ -1,10 +1,58 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players:
+        players_data = json.load(players)
+
+    for player_nickname, player_attributes in players_data.items():
+        race_name = player_attributes["race"]["name"]
+        if not Race.objects.filter(name=race_name).exists():
+            race = Race.objects.create(
```

Comment:
Use get_or_create
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create
Fix in all places